pkg/cli/cli_test.go

```diff
@@ -196,6 +196,7 @@ func Example_sql() {
 	// and with the query below the division by zero error will occur after the
 	// first batch consisting of 1 row has been returned to the client.
 	c.RunWithArgs([]string{`sql`, `-e`, `select 1/(@1-2) from generate_series(1,3)`})
+	c.RunWithArgs([]string{`sql`, `-e`, `SELECT '20:01:02+03:04:05' AS regression_65066`})
```

Comment:
oops sorry i missed before, but we'd need the `::timetz` cast here right? otherwise it is just a string value.

Reply:
![my bad](https://user-images.githubusercontent.com/3646147/118586263-c83c9100-b7dd-11eb-86b3-6ca43de2ac22.png)